low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -87,12 +87,15 @@ def __init__(self):
 
     def validate_server_type_support(self):
         """Check for supported server type."""
-        server_type = Conf.get(GLOBAL_CONF, NODE_TYPE_KEY).lower()
-        # TODO Add support for 'virtual' type server.
+        server_type = Conf.get(GLOBAL_CONF, NODE_TYPE_KEY)
```

Comment:
Provisioner supports following server types:
`VM, HW`

Reply:
Yes, covered the same.